tests/integration/src/xvm.rs

```diff
@@ -0,0 +1,334 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The file contains license and ownership information. No issues found.



<!-- This is an auto-generated comment by OSS CodeRabbit -->